src/styles/blocks/input.less

```diff
@@ -69,7 +70,7 @@ form {
         -moz-user-select: text;
         -ms-user-select: text;
         user-select: text;
-        max-width: calc(~"100% - "@labelMargin);
```

Comment:
This using for calculate checkbox with with label and use `text-overflow: elipsis` for label